tests/test_ci_gateway_bootstrap.py

```
@@ -40,3 +68,200 @@ def test_seed_credentials_copies_present_provider_keys_into_kv(monkeypatch) -> N
+    server = captured["server"]
+    thread = captured["thread"]
+    assert get_credential("OPENROUTER_API_KEY") == "router-secret"
+    assert get_credential(module.SERVER_AUTH_ENV_NAME) == "loopback-owner-token"
+    assert "OPENROUTER_API_KEY" not in module.os.environ
+    assert module.SERVER_AUTH_ENV_NAME not in module.os.environ
+    try:
+        port = server.server_address[1]
+        models_status, models_body = _request_json(
+            "GET",
+            f"http://127.0.0.1:{port}/v1/models",
+            token="loopback-owner-token",
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to argument: "token"

(S106)